frontend/src/components/pod/Details.tsx

```diff
@@ -116,6 +127,18 @@ function PodLogViewer(props: PodLogViewerProps) {
             ))}
           </Select>
         </FormControl>,
+        <FormControlLabel
```

Comment:
The option should only be made available only if the pod has restarted atleast once, else it will fail.